test/hotspot/jtreg/runtime/cds/appcds/javaldr/HumongousDuringDump.java

```diff
@@ -28,6 +28,7 @@
  * @library /test/lib /test/hotspot/jtreg/runtime/cds/appcds /test/hotspot/jtreg/runtime/cds/appcds/test-classes
  * @requires vm.cds.write.archived.java.heap
  * @requires vm.jvmti
+ * @requires vm.opt.final.UseCompressedOops == true
```

Comment:
Why is this change needed?

Reply:
I ran the test with the -XX:-UseCompressedOops and without the change and it passed. I've reverted the change.